src/plaid/plaidItem.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, ManyToOne } from 'typeorm'
+import { BaseModel, ModelId } from '../core/model'
+import { User } from '../users/user.entity'
+
+export const tableName = 'plaid_item'
+
+@Entity()
+export class PlaidItem extends BaseModel {
+  @Column()
+  itemId: string
+
+  @Column({ type: 'bytea' })
```

Comment:
just out of curiosity, why does this need a `bytea` type?

Reply:
The intent of this column is to store encrypted (raw) bytes.  Had tried `blob` and `mediumblob` data types but typeorm complained those weren't supported by postgresql: 

```
DataTypeNotSupportedError: Data type "mediumblob" in "PlaidItem.accessToken" is not supported by "postgres" database.
```

`bytea` was the first binary format in a quick googling that worked.  It automatically handles encoding the bytes into hex so we don't have to do that in code (e.g. no need to base64 encode the bytes).

Their docs have some info about text vs binary data columns: https://www.postgresql.org/docs/9.0/datatype-binary.html

"A binary string is a sequence of octets (or bytes). Binary strings are distinguished from character strings in two ways. First, binary strings specifically allow storing octets of value zero and other "non-printable" octets (usually, octets outside the range 32 to 126). Character strings disallow zero octets, and also disallow any other octet values and sequences of octet values that are invalid according to the database's selected character set encoding. Second, operations on binary strings process the actual bytes, whereas the processing of character strings depends on locale settings. In short, binary strings are appropriate for storing data that the programmer thinks of as "raw bytes", whereas character strings are appropriate for storing text."

Reply:
ha thanks for goolging it for me... i should just have checked....